docs/rpc/run-rpc-node-without-nearup.md

```diff
@@ -65,7 +65,7 @@ $ git fetch origin --tags
 Checkout to the branch you need if not `master` (default). Latest release is recommended. Please check the [releases page on GitHub](https://github.com/near/nearcore/releases).
 
 ```bash
-$ git checkout tags/1.25.0 -b mynode
+$ git checkout tags/1.31.0-rc.2 -b mynode
```

Comment:
Shouldn't this tag be `tags/1.31.0` too?

```suggestion
$ git checkout tags/1.31.0 -b mynode
```

Reply:
At the moment no, still in Rc2